modules/virt-using-windows-dvd-disk-image.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-automating-windows-sysprep.adoc
+
+[id="virt-using-windows-dvd-disk-image_{context}"]
+= Using a Windows DVD to create a VM disk image
+
+Microsoft does not provide disk images for download, but you can create a disk image using the Windows DVD. This disk image can then be used to create virtual machines.
+
+.Procedure
+
+. In the {VirtProductName} web console, click *Storage* -> *PersistentVolumeClaims* -> *Create PersistentVolumeClaim With Data upload form*.
+. Select the intended project (e.g. default to select the local windows DVD image).
+. Set the *Persistent Volume Claim Name* (e.g. win10-iso).
+. Upload the VM disk image from the Windows DVD.
+. The image is now available as a boot source to xref:../../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[create a new Windows VM].
```

Comment:
"The image is now available ..." is not a step. Better to add it to the previous step with a "+".